cartscript.js

```diff
@@ -0,0 +1,77 @@
+
+const cart=document.querySelector("cart")
+const cartRemove_btns= document.querySelectorAll(".trash-btn");
+addCartRemoveEvent();
+changingItemQuantityEvent();
+itemLikeEvent();
+
+///////// deleting item from a cart
+function addCartRemoveEvent(){
+    cartRemove_btns.forEach( function(btn) {
+        btn.addEventListener("click", handleRemoveCart);
+    })
+}
+
+///// changing quantity of an item in the cart
+function changingItemQuantityEvent(){
+    let cartQuantities= document.querySelectorAll(".cart-quantity");
+    cartQuantities.forEach(function(cartQuantity){
+        cartQuantity.addEventListener("change" , handle_ChangeItemQuantity);
+    })
+}
+
+//// clicking like button or heart Icon
+function itemLikeEvent(){
+    let heartBtns = document.querySelectorAll(".like-btn");
```

Comment:
`heartBtns` does not get reassigned thus it's better to defined using `const` here.